### PR TITLE
Use FocusEvent for testing for constructable events instead

### PR DESCRIPTION
**Pull request by arv**
Constructable FocusEvent was implemented after constructable MouseEvent.

### CODE REVIEW

File: src/wrappers/events.js [hunk 1]
```diff
@@ -481,7 +481,7 @@
 
   var supportsEventConstructors = (function() {
     try {
-      new window.MouseEvent('click');
+      new window.FocusEven('focus');
```
Comment (sethladd) [inline src/wrappers/events.js]: FocusEvent ?

Comment (dfreedm) [inline src/wrappers/events.js]: `FocusEvent` right?

Comment (arv) [inline src/wrappers/events.js]: Right. At least we know that the fallback path works now ;-)

On Mon, Nov 25, 2013 at 5:13 PM, Daniel Freedman
notifications@github.comwrote:

> In src/wrappers/events.js:
> 
> > @@ -481,7 +481,7 @@
> > 
> >    var supportsEventConstructors = (function() {
> >      try {
> > -      new window.MouseEvent('click');
> > -      new window.FocusEven('focus');
> 
> FocusEvent right?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/Polymer/ShadowDOM/pull/325/files#r7907390
> .

## 

erik

Comment (arv) [inline src/wrappers/events.js]: PTAL

On Mon, Nov 25, 2013 at 5:51 PM, Erik Arvidsson arv@chromium.org wrote:

> Right. At least we know that the fallback path works now ;-)
> 
> On Mon, Nov 25, 2013 at 5:13 PM, Daniel Freedman <notifications@github.com
> 
> > wrote:
> > 
> > In src/wrappers/events.js:
> > 
> > > @@ -481,7 +481,7 @@
> > > 
> > >    var supportsEventConstructors = (function() {
> > >      try {
> > > -      new window.MouseEvent('click');
> > > -      new window.FocusEven('focus');
> > 
> > FocusEvent right?
> > 
> > —
> > Reply to this email directly or view it on GitHubhttps://github.com/Polymer/ShadowDOM/pull/325/files#r7907390
> > .
> 
> ## 
> 
> erik

## 

erik

**Comment by arv** [comment]
Ping

**Comment by sorvell** [comment]
LGTM

**Pull request merged**